rpcserver.go

```diff
@@ -673,7 +673,6 @@ func (r *rpcServer) listBalancesByAsset(ctx context.Context,
 
 		resp.AssetBalances[assetIDStr] = &taprpc.AssetBalance{
 			AssetGenesis: &taprpc.GenesisInfo{
-				Version:      int32(balance.Version),
```

Comment:
Why are we removing this?

Reply:
Ah yeah this was left over. Basically this conflated the version of the tap commitment and the asset itself. IIRC, an assertion started to fail somewhere, but I think this was before I fixed the issue re max version tracking in the asset/tap commitment, will circle back then report.